apps/mivisonx_winml_classifier/README.md

```diff
@@ -0,0 +1,62 @@
+# Image Clasiification using AMD WinML Extension
+
+This application shows how to run supported ONNX models with MIVisionX RunTime:
```

Comment:
This application implements a method to run WinML supported ONNX models using MIVisionX RunTime. This c/c++ implementation does image classification on live camera capture or pre-recorded video stream. 

This application is a sample for developing windows application using MIVisionX WinML extension.

Reply:
resolve this